packages/cloudflare/src/api/get-cloudflare-context.ts

```diff
@@ -0,0 +1,49 @@
+import "server-only";
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  interface CloudflareEnv {}
+}
+
+export type CloudflareContext<
+  CfProperties extends Record<string, unknown> = IncomingRequestCfProperties,
+  Context = ExecutionContext,
+> = {
+  env: CloudflareEnv;
+  cf: CfProperties;
+  ctx: Context;
+};
+
+const cloudflareContextSymbol = Symbol.for("__cloudflare-context__");
+
+/**
+ * Utility to get the current Cloudflare context.
+ *
+ * Such context contains three values:
+ *  - `env`: object containing the worker's bindings
```

Comment:
nit: 
- move the doc to l12-14?
- use links `[]()`?

Reply:
Done 🙂 

Good suggestion regarding `[]()` I didn't know vscode would display md-style links correctly 🙂